### ISSUE TITLE
adding dependency old role andrewrothstein.zookeeper (v3.0.3)

### ISSUE THREAD

**Issue by patsevanton** [andrewrothstein/ansible-zookeeper-cluster#9]
ansible-galaxy install andrewrothstein.zookeeper-cluster
```
- adding dependency: andrewrothstein.zookeeper (v3.0.3)
- andrewrothstein.zookeeper (v3.0.3) was installed successfully
```

**Comment by andrewrothstein** [comment]
published zookeeper-cluster v3.2.1 which depends on zookeeper v3.0.12. the default zookeeper version is 3.6.2.

**Comment by andrewrothstein** [comment]
fixed bugs in the zookeeper role and pulled in latest. published zookeeper-cluster as v3.2.2.
